test/libsolidity/semanticTests/cleanup/cleanup_bytes_types_shortening_newCodeGen.sol

```diff
@@ -0,0 +1,17 @@
+contract C {
+    function f() public pure returns (bytes32 r) {
+        bytes4 x = 0xffffffff;
+        bytes2 y = bytes2(x);
+        assembly {
+            r := y
+        }
+        // At this point, r stores four bytes and y stores 2 bytes
+        // y is properly cleaned before the equality check
+        require(y == bytes2(0xffff));
```

Comment:
These three lines can actually be removed - the fact that `y` is already cleaned is apparent in the return value.